src/components/Header/BlockchainComp/index.tsx

```diff
@@ -123,24 +122,12 @@ export default memo(() => {
     ['node_version'],
     async () => {
       const { version } = await explorerService.api.fetchNodeVersion()
-      storeCachedData(AppCachedKeys.Version, `${version}&${new Date().getTime()}`)
+      cacheService.set<string>('node_version', version, { expire: ONE_DAY_TIMESTAMP })
```

Comment:
I think it is ok to use `cacheService.set('node_version', version, { expire: ONE_DAY_TIMESTAMP })` without the `<string>`.

Reply:
Yes, this suggestion is feasible from the perspective of code execution.

The main purpose of adding this is to provide more constraints to improve maintainability.

When this constraint exists, if the return value of `explorerService.api.fetchNodeVersion` changes due to development needs in the future, an error will be reported here to remind us to consider maintaining this code as well.